src/sentry/api/serializers/models/grouptombstone.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import absolute_import
+
+from itertools import izip
+import six
+
+from sentry.api.serializers import Serializer, register, serialize
+from sentry.constants import LOG_LEVELS
+from sentry.models import (
+    GroupTombstone, User
+)
+
+
+@register(GroupTombstone)
+class GroupTombstoneSerializer(Serializer):
+
+    def get_attrs(self, item_list, user):
+        project_set = set(i.project for i in item_list)
+        projects = {
+            p.id: d for p, d in izip(project_set, serialize(project_set, user))
```

Comment:
> Actually, would this not always be the same project for all GroupTombstones passed in, or do we want to account for potentially using this serializer organization-wide?

What's contained in the serializer response is really driven by the product requirements more than anything else, and my understanding is that any response will only contain tombstones from the same project at the moment. Personally I'd remove the project data for now (and it's easy enough to add it back if or when it's needed.)